modules/dkan/dkan_datastore/src/Manager/Manager.php

```diff
@@ -235,6 +245,7 @@ public function import() {
     if ($import_state === self::DATA_IMPORT_DONE) {
       $this->stateDataImport = self::DATA_IMPORT_DONE;
       $this->saveState();
+      module_invoke_all('datastore_post_import', $this->resource);
```

Comment:
@dharizza do we still need this?

Reply:
Yes we do, because that's what will allow us to execute custom actions after a datastore import is completed.